.gitignore

```diff
@@ -0,0 +1,66 @@
+# Ignore docs files
+
+# Compiled source #
+###################
+*.com
+*.class
+*.dll
+*.exe
+*.o
+*.so
+*.nupkg
+
+# Packages #
+############
+# it's better to unpack these files and commit the raw source
+# git has its own built in compression methods
+*.7z
+*.dmg
+*.gz
+*.iso
+*.jar
+*.rar
+*.tar
+*.zip
+
+# Numerous always-ignore extensions
+*.diff
+*.err
+*.log
+*.orig
+*.rej
+*.swo
+*.swp
+*.vi
+*.bak
+*~
+
+# OS or Editor folders
+._*
+.cache
+.DS_Store
+.idea
+.replace
+.compiled_html
+.project
+.settings
+.tmproj
+*.esproj
+*.sublime-project
+*.sublime-workspace
+nbproject
+Thumbs.db
+
+# grunt-html-validation
+validation-report.json
+validation-status.json
+
+# Folders to ignore
+bower_components
+node_modules
+no_git
+
+# Special files
+*.ftpauth
+
+
```

Comment:
I already have an hidden .gitignore with custom configs.

No need of that .gitignore.